### PR TITLE
Add option to specify different sources per ROS distro

### PR DESCRIPTION
**Pull request by Timple**
Might be useful when switching to melodic in the future but when not all packages are available yet:
http://repositories.ros.org/status_page/compare_kinetic_melodic.html

### CODE REVIEW

File: installer/parse-install-yaml.py [hunk 1]
```diff
@@ -31,19 +32,32 @@ def main():
 
         try:
             install_type = install_item["type"]
-            if install_type == "ros":
-                source_type = install_item["source"]["type"]
+            if install_type == "empty":
```
Comment (MatthijsBurgh) [inline installer/parse-install-yaml.py]: We need to merge the empty target PR first, which has some unresolved issues.

File: installer/parse-install-yaml.py [hunk 2]
```diff
@@ -31,19 +32,32 @@ def main():
...
+                return 0
+            elif install_type == "ros":
+                if "source" in install_item:
+                    source = install_item["source"]
+                else:
+                    release = environ["ROS_DISTRO"]
```
Comment (MatthijsBurgh) [inline installer/parse-install-yaml.py]:
```suggestion
                    release = environ["TUE_ROS_DISTRO"]
```
More consistent with rest of the installer

File: installer/parse-install-yaml.py (hunk 1 above)
Comment (Timple) [reply installer/parse-install-yaml.py]: Aaah, that makes sense. I was actually wondering where the Empty statement went anyway.

File: installer/parse-install-yaml.py (hunk 2 above)
Comment (Timple) [reply installer/parse-install-yaml.py]: Suits me, but in which case are they different?

Comment (MatthijsBurgh) [reply installer/parse-install-yaml.py]: I don’t think they will be different at any time. But TUE_ROS_DISTRO, will probably be set before ROS_DISTRO, so definitely always available.

**Pull request merged**